Docs/ProjectSpec.md

```diff
@@ -251,12 +251,14 @@ A source can be provided via a string (the path) or an object of the form:
 	- `sources` - Compile Sources phase
 	- `resources` - Copy Bundle Resources phase
 	- `headers` - Headers Phase
+	- `copyFiles` - Copy Files Phase
 	- `none` - Will not be added to any build phases
+- [ ] **copyFiles**: **[Copy Files](#copy-files)** - Options for the Copy Files build phase. If specified the `buildPhase` needs to be `copyFiles` or unspecified (in which case it will default to `copyFiles`).
```

Comment:
Instead of this I could make it so when you specify `copyFiles` for `buildPhase` you have to do it as an object:

```yml
- buildPhase:
    type: copyFiles
    destination: productsDirectory
    subpath: include/$(PRODUCT_NAME)
```

This might complicate it too much though, which is why I went the route I did.

Reply:
Also, I believe the way people will specify this is just by using the `copyFiles` object and not specifying a `buildPhase` manually.